includes/admin/class-amp-cache-utilities.php

```diff
@@ -0,0 +1,251 @@
+<?php
+abstract class AMP_Cache_Utilities {
```

Comment:
It doesn't have to be, but this way, someone won't accidentally instantiate multiple copies of it wastefully.